src/state/ducks/student/sagas.ts

```diff
@@ -1,16 +1,36 @@
 import { call, put, takeLatest } from 'redux-saga/effects';
 import apiCaller from '../../utils/apiCaller';
 import { IStudent, StudentActionTypes } from './types';
-import { handleFetchStudentsAction } from './reducers';
+import { handleFetchStudentsAction, handleOneStudentAction } from './reducers';
+
+function* handleAddStudent(data: any): Generator {
+  const addStudent = (yield call(apiCaller, 'POST', 'students', data.meta.data)) as IStudent;
+  yield put(handleFetchStudentsAction(addStudent));
+}
 
 function* handleFetchStudents(): Generator {
   const students = (yield call(apiCaller, 'GET', 'students')) as IStudent[];
 
   yield put(handleFetchStudentsAction(students));
 }
 
+function* handleOneStudent(data: any): Generator {
+  const oneStudent = (yield call(apiCaller, 'GET', `students/${data.meta.data}`)) as IStudent;
+
+  yield put(handleOneStudentAction(oneStudent));
+}
+
+function* handleEditStudents(data: any): Generator {
+  const editstudents = (yield call(apiCaller, 'PUT', `students/${data.meta.meta}`), data.meta.data) as IStudent[];
```

Comment:
what is meta.meta here ?
Does not seems to be right. Have you checked the working for this.

Reply:
no it was in the previous push. now it's data.meta.data